apps/meteor/server/models/raw/LivechatVisitors.ts

```diff
@@ -159,11 +160,16 @@ export class LivechatVisitorsRaw extends BaseRaw<ILivechatVisitor> implements IL
 	/**
 	 * Find visitors by their email or phone or username or name
 	 */
-	findPaginatedVisitorsByEmailOrPhoneOrNameOrUsername(
+	async findPaginatedVisitorsByEmailOrPhoneOrNameOrUsernameOrCustomField(
 		emailOrPhone: string,
 		nameOrUsername: RegExp,
 		options: FindOptions<ILivechatVisitor>,
-	): FindPaginated<FindCursor<ILivechatVisitor>> {
+	): Promise<FindPaginated<FindCursor<ILivechatVisitor>>> {
+		const allowedCF = LivechatCustomField.find({ scope: 'visitor', searchable: true }, { projection: { fields: { _id: 1 } } }).map(
```

Comment:
`projection` is the replacement of `fields`. So, change this to:
```suggestion
		const allowedCF = LivechatCustomField.find({ scope: 'visitor', searchable: true }, { projection: { _id: 1 } }).map(
```